account_usability/models/account_group.py

```diff
@@ -1,6 +1,7 @@
 # Copyright 2018 FOREST AND BIOMASS ROMANIA SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
+
 from odoo import fields, models
```

Comment:
Nitpicking : useless

Reply:
Sorry, do you mean the accidently added extra newline?

Reply:
I guessed it so I removed it anyway, it is better to remove for cleaner history anyways :)